packages/@dcl/sdk-commands/src/commands/quests/linker-dapp/routes.ts

```diff
@@ -0,0 +1,68 @@
+import { dirname, resolve } from 'path'
+import { Router } from '@well-known-components/http-server'
+import { IFuture } from 'fp-future'
+
+import { CliComponents } from '../../../components'
+import { CreateQuest, QuestLinkerActionType } from '../types'
+import { LinkerResponse, getContentType } from '../../../linker-dapp/api'
+
+export function setRoutes(
```

Comment:
why not reuse the setRoutes that we currently use for the other dapps ?
It seems that its the same routes and logic, and if we need to add some new one, you could extend it since setRoutes returns the route variable